### ISSUE TITLE
The config options don't save properly

### ISSUE THREAD

**Issue by RandomGuy91** [henkelmax/corpse#189]
### Bug description

When I change the config options using the Configured mod and save it, the changes dont save. I looked in the config folder and there is no file for Corpse, but there is obviously a way for Configured to be dynamically generating the config menu. 

There is a line in the log that says I started and stopped editting it but it doesn't save it:
20:07:14.068       Configured        Render thread        Started editing 'corpse-server.toml'
20:07:14.068       Configured        Render thread        Stopped editing 'corpse-server.toml'

### Minecraft version

1.20.1

### Mod version

1.0.12

### Mod loader and version

Forge 1.0.12

### Steps to reproduce

Try and edit the config file for corpse with the configured mod and save.

### Expected behavior

I expected the changes to be saved, but they weren't

### Log files

There are not any relevant log details besides the ones I put at the top.

**Comment by henkelmax** [comment]
Thats an issue of the Configured  mod then. Please check if this also happens when manually editing the config file. If it still happens then, open another ticket.

**Comment by RandomGuy91** [comment]
There is no config file in the config folder or defaultconfig folder

**Comment by henkelmax** [comment]
Yes. Server configs are in the world folder. Read the FAQ for more information.

**Comment by RandomGuy91** [comment]
Ah, I see. Sorry about that.
